src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -351,31 +349,31 @@
         "ENABLE_ACCOUNT": "Activer le compte",
         "DELETE": "Supprimer",
         "CREAT_ROBOT_ACCOUNT": "Création d'un compte robot",
-        "PERMISSIONS_ARTIFACT": "Artéfact",
-        "PERMISSIONS_HELMCHART": "Helm Chart (Chart Museum)",
-        "PUSH": "Pousser",
-        "PULL": "Tirer",
-        "FILTER_PLACEHOLDER": "Filtrer Robot Accounts",
-        "ROBOT_NAME": "Ne peut pas contenir de caractères spéciaux(~#$%) et la longueur maximale devrait être de 255 caractères.",
+        "PERMISSIONS_ARTIFACT": "Artefact",
+        "PERMISSIONS_HELMCHART": "Chart Helm (Chart Museum)",
+        "PUSH": "Push",
+        "PULL": "Pull",
+        "FILTER_PLACEHOLDER": "Filtrer les comptes robot",
+        "ROBOT_NAME": "Ne peut pas contenir de caractères spéciaux(~#$%) et la longueur maximale doit être de 255 caractères.",
         "ACCOUNT_EXISTING": "Le compte robot existe déjà.",
         "ALERT_TEXT": "C'est le seul moment pour copier votre jeton d'accès personnel. Vous n'aurez pas d'autre opportunité",
         "CREATED_SUCCESS": "Création de '{{param}}' effectuée.",
         "COPY_SUCCESS": "Copie de '{{param}}' effectuée",
-        "DELETION_TITLE": "Confirmer la suppression des comptes du robot",
-        "DELETION_SUMMARY": "Voulez-vous supprimer les comptes du robot {{param}}?",
+        "DELETION_TITLE": "Confirmer la suppression des comptes robot",
+        "DELETION_SUMMARY": "Voulez-vous supprimer les comptes robot {{param}} ?",
         "PULL_IS_MUST": "La permission 'Pull' est activée par défaut et ne peut pas être modifiée.",
         "EXPORT_TO_FILE": "Exporter vers un ficher",
         "EXPIRES_AT": "Expire le",
         "EXPIRATION_TOOLTIP": "Si non renseigné, le délai d'expiration système sera utilisé",
-        "INVALID_VALUE": "La valeur du moment d'expiration est invalide",
+        "INVALID_VALUE": "La valeur de la date d'expiration est invalide",
         "NEVER_EXPIRED": "Ne jamais expirer",
-        "NAME_PREFIX": "Préfixe du Nom de Robot",
-        "NAME_PREFIX_REQUIRED": "Le Préfixe du Nom de Robot est obligatoire"
+        "NAME_PREFIX": "Préfixe du nom du compte robot",
+        "NAME_PREFIX_REQUIRED": "Le préfixe du nom du compte robot est obligatoire"
     },
     "WEBHOOK": {
-        "EDIT_BUTTON": "EDITER",
-        "ENABLED_BUTTON": "ACTIVER",
-        "DISABLED_BUTTON": "DESACTIVER",
+        "EDIT_BUTTON": "Editer",
```

Comment:
```suggestion
        "EDIT_BUTTON": "Éditer",
```

Reply:
In French it's tolerated to have upper case without accent, but ok